proto/src/einride/iam/example/v1/freight_service.proto

```diff
@@ -346,6 +351,17 @@ service FreightService {
     };
   }
 
+  option (einride.iam.v1.operations_permissions) = {
+    operation_permissions {
```

Comment:
Mismatch in naming, I think. We should probably either change to `einride.iam.v1.operation_permissions` or to `operations_permissions { ... }`.

Reply:
I updated naming to make the top-level more explicitly named `longrunning_operations_permissions` - the second level is permissions for a single operation - that's why operation is singular here!

Reply:
Perfect!